localization/english/event_localization/wc_demon_hunter_events_l_english.yml

```
@@ -0,0 +1,62 @@
+ demon_hunter_0005_A:0 "This is just the beginning, I see..."
+ demon_hunter_0005_A_tt:0 "#help You get access to a decision which allows you to attain the [GetTrait('metamorphosis').GetName(GetNullCharacter)] form.#!"
+
+ demon_hunter_0006_title:1 "A Destiny of Sorrow"
+ demon_hunter_0006_desc:0 "Time has passed. Holding onto the [GetTrait('metamorphosis').GetName(GetNullCharacter)] form comes with a strain on the body.\n\nDo I #italic need #!to stay in this form longer? Or do I let go and let my body rest and come back stronger later?"
+ demon_hunter_0006_A:0 "I will be cautious"
+ demon_hunter_0006_B:0 "I can tolerate it..."
+ 
+ demon_hunter_0007_title:1 "A Destiny of Sorrow"
+ demon_hunter_0007_desc:0 "I have stayed in this form for long... the side effects are starting to set in. I hear strange voices and sometimes noises.\n\nHeadaches are setting in and occasionally I get enraged over small issues."
+ demon_hunter_0007_A:0 "No use risking it any further"
+ demon_hunter_0007_B:0 "I have lasted this long..."
+
+ demon_hunter_0008_title:1 "A Destiny of Sorrow"
+ demon_hunter_0008_desc:0 "The power that has burnt inside me for this long now feels like a sickness. My hunter markings burn to try and keep that [GetTrait('being_demon').GetName(GetNullCharacter)] at bay.\n\nDark urges and fantasies cloud my mind."
+ demon_hunter_0008_A:0 "This has gone on for too long"
+ demon_hunter_0008_B:0 "... I can last more"
+
+ demon_hunter_0009_title:1 "A Destiny of Sorrow"
+ demon_hunter_0009_desc:0 "The arcane hunter markings burn on my body as brightly as ever. My gamble has lasted for long but it might not anymore... \n\n#XB "Tired so soon?"#! , I hear [first_demon_prey.GetFirstName] say. #XB "Come now, Hunter. You know this is just the beginning!... You know these are not the limits of the powers I can provide... Think about how many will suffer if you don't continue this crusade... Think about how many of your allies might die if you don't #italic sacrifice #!..."#!"
+ demon_hunter_0009_A:0 "Time to abandon this insanity!"
+ demon_hunter_0009_B:0 "... Just... one last time..."
+
+ keep_the_form_going_tt:0 "You keep the [GetTrait('metamorphosis').GetName(GetNullCharacter)] form."
+
+ demon_takes_over_demon_hunter_tt:0 "#XB The [GetTrait('being_demon').GetName(GetNullCharacter)] within takes over your soul and being#!"
+
+ demon_hunter_0010_title:1 "A Destiny at End"
+ demon_hunter_0010_desc:0 "The [GetTrait('being_demon').GetName(GetNullCharacter)] contained within [new_fallen_hunter.GetFirstName] is no longer imprisoned. [first_demon_prey.GetFirstName]'s and [new_fallen_hunter.GetFirstName]'s souls have merged and any memory of either of their pasts is now gone. #help Will you play as the [GetTrait('fallen_demon_hunter').GetName(GetNullCharacter)]? Or as [player_heir.GetFirstName] and avenge [new_fallen_hunter.GetFirstName]?#!"
+ demon_hunter_0010_A:0 "Fate of the Fallen Hunter"
+ demon_hunter_0010_B:0 "Fate of the Hunter's Cub"
```

can we change these so that they are "in character" rather than just option labels? this feels very "game-y" in ways that CK3 events usually dont

---

I am changing and making a different description for when a character is alone and doesnt have an heir, this will be done in that.

But these options and this event itself will stay as is. `demon_hunter_0010` is *supposed* to be just a notifications event because i wasn't even gonna implement this event before.

But after the "sudden death", an event was needed as to what is happening/haven't to the player character.

---

Can i just suggest a small change then? 
```suggestion
 demon_hunter_0010_A:0 "Choose the fate of the Fallen Hunter"
 demon_hunter_0010_B:0 "Choose the fate of the Hunter's Cub"
```